### PR TITLE
Enable signle-line rows creation for horizontal mode

### PR DESCRIPTION
**Pull request by ialimz**
Roadmap: wrap TagListView into UIScrollView

### CODE REVIEW

**Comment by Cee** [comment]
Could you tell us why these changes are necessary?

File: TagListView/TagListView.swift [hunk 1]
```diff
@@ -187,7 +196,13 @@ open class TagListView: UIView {
             rearrangeViews()
         }
     }
-    
+
```
Comment (taratandel) [inline TagListView/TagListView.swift]: when you rearrange subView you should consider the alignment too, we the property `wrapTagsToNextRow` is set to `false` the alignment is always right

File: TagListView/TagListView.swift [hunk 2]
```diff
@@ -279,23 +284,27 @@ open class TagListView: UIView {
...
-    override open var intrinsicContentSize: CGSize {
+
+    open override var intrinsicContentSize: CGSize {
         var height = CGFloat(rows) * (tagViewHeight + marginY)
         if rows > 0 {
             height -= marginY
         }
-        return CGSize(width: frame.width, height: height)
+        var width = frame.width
+        if !wrapTagsToNextRow {
```
Comment (taratandel) [inline TagListView/TagListView.swift]: when you rearrange subView you should consider the alignment too, we the property `wrapTagsToNextRow` is set to `false` the alignment is always right

Comment (ialimz) [reply TagListView/TagListView.swift]: Thanks Tara. We now use CollectionView for handling tags. Its more reliable and customizable

**Pull request closed without merging**